### PR TITLE
Eq 1443 updates

### PR DESCRIPTION
**Pull request by curlymorphic**
Updated the Eq plugin as per #1443 

made shelving filters stable,
removed oversampling,
The input and output analysers are now independently switchable, using the labels in the widget
While the mouse has focus in the widget, the current frequency for that location is displayed in a TextFloat,
Peaking filters now use bandwidth instead of resonance.

### CODE REVIEW

File: plugins/Eq/DBvModel.cpp [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#include "DBvModel.h"
+
+DBvModel::DBvModel(float val, float min, float max, float step,
+								   Model *parent, const QString &displayName,
+								   bool defaultConstructed) :
+		FloatModel( val, min, max, step, parent, displayName, defaultConstructed )
+{
+		connect(this, SIGNAL( dataChanged() ), this,SLOT( calcAmp() ) );
```
Comment (diizy) [inline plugins/Eq/DBvModel.cpp]: Bad idea to use signals for this. Causes more overhead and possible latency. You could look at how I've done the dBV to linear conversion in CrossoverEq: by using the isValueChanged() method, you can remove the need to use signals.

**Comment by diizy** [comment]
I'll merge this now but you might want to look into the signals thing... in general, in DSP contexts it's best to avoid signals when possible.

**Comment by curlymorphic** [comment]
I have been working on a patch to change this, removing the DBvModels and just checking every period if the model value has changed, the same as in CrossoverEq. Do you still want it ? (just got to test it first)

**Comment by diizy** [comment]
On 01/04/2015 06:24 PM, Dave wrote:
>
> I have been working on a patch to change this, removing the DBvModels
> and just checking every period if the model value has changed, the
> same as in CrossoverEq. Do you still want it ? (just got to test it first)
>

Yeah, sure. Optimizations are always good.

**Pull request merged**